pages/Source_Code_Analysis_Tools.md

```diff
@@ -4,47 +4,63 @@ layout: col-sidebar
 
-Some tools are starting to move into the IDE. For the types of problems that can be detected during the software development phase itself, this is a powerful phase within the development life cycle to employ such tools, as it provides immediate feedback to the developer on issues they might be introducing into the code during code development itself. This immediate feedback is very useful, especially when compared to finding
-vulnerabilities much later in the development cycle.
+You can add some SAST tools into your IDE. Such tools can help you detect issues during software development.
+SAST tool feedback can save time and effort, especially when compared to finding vulnerabilities later in the development cycle.
 
 ## Strengths and Weaknesses
 
 ### Strengths
 
 - Scales well -- can be run on lots of software, and can be run repeatedly (as with nightly builds or continuous integration).
-- Useful for things that such tools can automatically find with high confidence, such as buffer overflows, SQL Injection Flaws, and so forth.
-- Output is good for developers -- highlights the precise source files, line numbers, and even subsections of lines that are affected.
+- Identifies certain well-known vulnerabilities, such as:
+  - Buffer overflows
+  - SQL Injection Flaws
+- Output helps developers, as SAST tools highlight the problematic code, by filename,
+  location, line number, and even the affected code snippet.
 
 ### Weaknesses
 
-- Many types of security vulnerabilities are difficult to find automatically, such as authentication problems, access control issues, insecure use of cryptography, etc. The current state of the art only allows such tools to automatically find a relatively small percentage of application security flaws. However, tools of this type are getting better.
+- Difficult to automate searches for many types of security vulnerabilities, including:
+  - Authentication problems
+  - Access control issues
+  - Insecure use of cryptography
+  Current SAST tools are limited. They can automatically identify only a relatively
+  small percentage of application security flaws.
 - High numbers of false positives.
-- Frequently can't find configuration issues, since they are not represented in the code.
+- Frequently unable to find configuration issues, since they are not represented in the code.
 - Difficult to 'prove' that an identified security issue is an actual vulnerability.
-- Many of these tools have difficulty analyzing code that can't be compiled. Analysts frequently can't compile code because they don't have the right libraries, all the compilation instructions, all the code, etc.
+- Many SAST tools have difficulty analyzing code that can't be compiled.
+  - Analysts frequently cannot compile code unless they have:
+    - Correct libraries
+    - Compilation instructions
+    - All required code
```

Comment:
I'm not convinced that any of this detail is needed. It's somewhat redundant to "analyzing code that can't be compiled".